### ISSUE TITLE
Fine-tuning

### ISSUE THREAD

**Issue by bornTalented** [TensorSpeech/TensorFlowASR#102]
How do I finetune subword conformer?

**Comment by usimarit** [comment]
You can change the `config.yml` file to whatever you want the model to be.

**Comment by bornTalented** [comment]
Thanks for your reply. But I am unable to find the configuration in the `config.yml` file that will load the pretrained model for finetuning.
Could you please help me by providing an example.

**Comment by usimarit** [comment]
You can create a script like [train_subword_conformer.py](https://github.com/TensorSpeech/TensorFlowASR/blob/main/examples/conformer/train_subword_conformer.py) to load the pretrained model.
If you need deeper fine-tuning like freezing layers, then just copy the [conformer.py](https://github.com/TensorSpeech/TensorFlowASR/blob/main/tensorflow_asr/models/conformer.py) to a separate file (`my_conformer.py` for example), then edit that file, or edit directly on `conformer.py` file. Then import the new model to the script you created for loading pretrained weights by name and reinstall the package.

**Comment by bornTalented** [comment]
As mentioned I have changed in the train_subword_conformer.py file to load weights from pretrained model. But after finetuning model is producing short outputs in prediction.

```
GROUNDTRUTH	
i have committed a great sin by killing my friend.	i have to 
there cannot be a greater sin than killing a trusted friend.	agree that 
GREEDY (prediction)
i have to 
agree that 

```
could you help me by explaining the reason behind that?

**Comment by usimarit** [comment]
@bornTalented What did the loss look like after finetuning?

**Comment by usimarit** [comment]
I did some fine-tuning and the model converges around 2-3 epochs, so I'll close this issue here 😄
